### ISSUE TITLE
libui

### ISSUE THREAD

**Issue by markand** [kozross/awesome-c#122]
The [libui](https://github.com/andlabs/libui) project consists of a native looking GUI toolkit for C. I have not tested, reviewed yet.

**Comment by kozross** [comment]
I've taken a look at libui, but I'm reluctant to include it. By the author's own admission, the project is 'mid-alpha', and is _deliberately_ not documented (again, by the author's own admission). If you think it deserves inclusion, I'm willing to listen, but from my view, it's a bit too early for it.

**Comment by markand** [comment]
Yes I've felt the same once I dig into it. It's not ready for inclusion yet so should we close the issue or we can keep it open with some special flag to never forget to stay tuned with this library?

**Comment by kozross** [comment]
I think we can leave it open - I've made a new tag just for this purpose. Thanks for bringing this library to my attention!
